include/extractor/extraction_way.hpp

```diff
@@ -114,6 +115,7 @@ struct ExtractionWay
     bool is_startpoint : 1;
     bool forward_restricted : 1;
     bool backward_restricted : 1;
+    bool is_left_hand_driving : 1;
```

Comment:
after some hours of debugging i think i will keep bool type here, because of some strange gcc behavior here: with `std::uint8_t` setting `is_startpoint` also sets `forward_restricted` and `backward_restricted`. it can be either ub or gcc optimization issue